db/seeds.rb

```diff
@@ -1,3 +1,5 @@
+ActiveFedora.fedora.connection.send(:init_base_path)
```

Comment:
Is this something that should be outside of this commit?

Reply:
i think it's tightly motivated by this environment---other existing environments seem to provide the base path for us(?).

i'm not sure moving it to a separate PR is helpful, since that will move it further away from its motivation. but maybe there are better approaches for the app to ensure its base path is present? maybe it's a whole different rake task that can be called in a more targeted way?

Reply:
I'm satisfied with the answer.  Keep this as part of the commit.